cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -704,7 +704,8 @@ CruiseControlReconciler cruiseControlReconciler()   {
                     kafkaBrokerNodes,
                     kafkaBrokerStorage,
                     kafkaBrokerResources,
-                    clusterCa
+                    clusterCa,
+                    new PasswordGenerator(16)
```

Comment:
I think we already have a password generator.

```suggestion
                    passwordGenerator
```

Reply:
Right. Fixed.